.flake8

```diff
@@ -60,14 +60,32 @@ max-line-length = 100
 
 # Allow certain violations in certain files:
 per-file-ignores =
-  docs/conf.py: WPS221, WPS226, WPS301, WPS305, WPS323, WPS420, WPS429, WPS433
+  # The following ignores have been researched and should be considered permanent
+  # each should be preceeded with an explanation of each of the error codes
+  # These might be commented out as they overlap with the perfile ignores
+  # once the per file ignores are removed or added within the file itself
+  # they should be able to be uncommented
+
+  # S101: Allow the use of assert within the tests directory
+  # tests/*: S101
+
+  # The following were present during the initial implemetation
+  # Please try to keep it in alphbetical order to make finding files easier for the next person
+
   docs/_ext/single_sourced_data.py: D400, D403, DAR101, DAR201, N817, P103, WPS110, WPS111, WPS201, WPS202, WPS210, WPS213, WPS221, WPS229, WPS231, WPS237, WPS305, WPS318, WPS323, WPS331, WPS336, WPS347, WPS407, WPS432, WPS440, WPS441, WPS453, WPS504
   docs/_ext/spelling_stub_ext.py: DAR101, DAR201
+  docs/conf.py: WPS221, WPS226, WPS301, WPS305, WPS323, WPS420, WPS429, WPS433
```

Comment:
This doesn't look related to asserts or tests. So I'd argue that this part should be submitted separately because it doesn't belong to the atomic effort of setting up the ignores for the tests.

Reply:
I'll work with whomever originally implemented the list to see if it can be sorted correctly.

Reply:
See: https://github.com/ansible/ansible-navigator/pull/794